src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/index.js

```diff
@@ -1,8 +1,8 @@
 import { connect } from 'react-redux';
 import DocumentItemsComponent from './DocumentItems';
 
-const mapStateToProps = state => ({
-  documentDetails: state.documentDetailsFront.data,
+const mapStateToProps = (state, ownProps) => ({
+  documentDetails: ownProps.documentDetails || state.documentDetailsFront.data,
```

Comment:
did you check if it does not break the DocumentDetails page of standalone `react-invenio-app-ils`?

Reply:
I checked, it doesn't break it.